src/sentry/api/serializers/models/team.py

```diff
@@ -184,3 +185,22 @@ def serialize(
         d["projects"] = attrs["projects"]
         d["externalTeams"] = attrs["externalTeams"]
         return d
+
+
+class TeamSCIMSerializer(Serializer):  # type: ignore
+    def serialize(
```

Comment:
Are you ever going to be serializing more than one team at a time here? If so, it might make sense to do this in `get_attrs` so that we don't have to fetch the same org member multiple times.